Gems/Atom/Feature/Common/Assets/Materials/Types/EnhancedPBR.materialtype

```diff
@@ -1223,14 +1223,53 @@
                     "min": 0.0,
                     "softMax": 20.0
                 },
+                {
+                    "name": "shrinkFactor",
+                    "displayName": "    Shrink Factor",
+                    "description": "Shrink (absolute) offset towards the normal opposite direction to ensure correct shadow map projection",
+                    "type": "float",
+                    "defaultValue": 0.005,
+                    "min": 0.0,
+                    "softMax": 0.05,
+                    "connection": {
+                        "type": "ShaderInput",
+                        "name": "m_shrinkFactor"
+                    }
+                },
+                {
+                    "name": "transmissionAngleBias",
+                    "displayName": "    Angle Bias",
+                    "description": "cosine of angle to extend below (N . L = 0) in scattering through thin objects",
+                    "type": "float",
+                    "defaultValue": 0.1,
+                    "min": -1.0,
+                    "softMax": 1.0,
+                    "connection": {
+                        "type": "ShaderInput",
+                        "name": "m_transmissionNdLBias"
+                    }
+                },
+                {
+                    "name": "distanceAttenuation",
+                    "displayName": "    Distance Attenuation",
+                    "description": "Attenuation applied to hide artifacts due to low-res shadow maps (e.g. objects far to the camera when using directional lights or objects far to the light when using sphere/disk lights",
+                    "type": "float",
+                    "defaultValue": 4.0,
+                    "min": 0.0,
+                    "softMax": 10.0,
```

Comment:
nit: this might be a tad too close, especially for larger objects like sheet / hut cover (however it's a SoftMax, so the user can force it)

Reply:
In this case the higher the value the higher the attenuation with distance will be (it acts as a factor). Therefore, the factor can be 0 (or close to 0) for no attenuation no matter how far we are.  
Maybe I understood wrong... which is your suggestion?